docker-compose.yml

```diff
@@ -3,11 +3,10 @@ services:
   app: &main
     build: .
     image: keboola/python-transformation-v2
+    user: www-data
```

Comment:
Aj v `docker-compose` to chceme testovat s `www-data` userom, kedze s tym boli problemy, ale testy prechadzali, lebo bezali pod `root`.